lib/glare/cf_dns_records.rb

```diff
@@ -25,32 +40,23 @@ def count
       @records.count
     end
 
+    def contents
+      @records.map(&:content)
+    end
+
     def each
       @records.each { |record| yield(record) }
     end
 
-    def records_to_delete(targer_number)
-      records_to_delete = count - targer_number
-      return [] if records_to_delete < 0
+    def records_to_delete(target_number)
+      records_to_delete = count - target_number
+      return CfDnsRecords.new([]) if records_to_delete < 0
```

Comment:
Have you considered creating this empty collection in a factory method too? E.g. ``CfDnsRecords.empty``